code/Common/Maybe.h

```diff
@@ -82,7 +82,7 @@ struct Maybe {
     Maybe(const Maybe &) = delete;
 
 private:
-    T _val;
+    T _val = false;
```

Comment:
@kimkulling Wrong line. the thing below it should be default initialized.

Reply:
Saw it, already fixed. Thanks for the hint!